css/style.css

```diff
@@ -0,0 +1,164 @@
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.container {
+  width: 960px;
+  margin: auto;
+}
+
+header {
+  height: 150px; 
+  background-color: #000;
+  width: 100%;
+  border-bottom: 7px solid yellow;
+}
+
+header h1 {
+  color: yellow;
+  float: left;
+  padding: 20px 0 0 20px;
+  margin-top: 0px;
+  font-family: georgia;
+}
+
+.topnav {
+  float: right;
+}
+
+.topnav a {
+  color: white;
+  font-size: 24px;
+  text-decoration: none;
+  padding: 10px;
+}
+
+.impact {
+  font-family: impact;
+  letter-spacing: 1px;
+}
+
+.geneva {
+  font-family: 'geneva';
+  letter-spacing: 0;
+}
+
+.subhead {
+  background-color: #D3D3D3;
+  text-align: center;
+  font-style: italic;
+  font-family: georgia;
+  padding: 10px;
+  font-size: 20px;
+  border-top: 10px yellow;
+}
+
+#strike {
+  color: grey;
+  text-decoration: line-through;
+}
+
+#underline {
+  border-bottom: 2px yellow solid;
+}
+
+.main {
+  background: url("../images/coworking.jpg") no-repeat center;
+  float: right;
+  width: 100%;
+  height: 445px
```

Comment:
try avoid setting heights.
Let the child elements automatically stretch the parent :)